src/packages/steps/steps.harmony.css

```
@@ -1,8 +1,158 @@
+  background-color: #ff0f23;
+  border-radius: 50%;
+  opacity: 0.23;
+}
+
+.nut-steps-horizontal {
+  display: flex;
+}
+.nut-steps-horizontal .nut-step {
+  flex-grow: 0;
+  flex-shrink: 0;
+  flex: 1;
```

Biome

[error] Unexpected shorthand property flex after flex-grow



(lint/suspicious/noShorthandPropertyOverrides)